src/components/templates/Jazzbar/JazzTab/JazzTab.tsx

```diff
@@ -195,12 +194,7 @@ const Jazz: React.FC<JazzProps> = ({ venue }) => {
                       key="main-event"
                       title="main event"
                       className="iframe-video"
-                      src={
-                        convertToEmbeddableUrl({
-                          url: venue.iframeUrl,
-                          autoPlay: venue?.autoPlay,
-                        }) ?? ""
-                      }
+                      src={iframeUrl}
```

Comment:
is it possible to make them play immediately? because in my venue I have to start them manually.